scripts/superdesk-monitoring/aggregate.js

```diff
@@ -381,6 +381,13 @@
             });
         };
 
+        this.reload = function() {
+            $timeout(function() {
+                self.search(self.searchQuery);
+            }, 1, false);
+            self.search();
```

Comment:
Thanks for explaining that. I've just checked out your branch and I've created an item in the personal space, but I can see it in the "Activity Stream" widget using the `admin` user. Am I doing something wrong?